frontends/p4/fromv1.0/v1model.h

```diff
@@ -307,6 +307,9 @@ class V1Model : public ::Model::Model {
     const DirectCounter_Model directCounter;
 
     static V1Model instance;
+    // The following match constants appearing in v1model.p4
+    static const char* versionInitial;  // 20180101
+    static const char* version2020;     // 20200408
```

Comment:
This should probably be `versionCurrent` (or even just `version` -- not sure anyone needs other than the current version?)  This is being used primarily to get the version number to dump as a #define when we dump the IR as P4_16 code.

Reply:
It is the current one, but if we add further variants suddenly old programs will stop working if current is always the latest.
I have come to appreciate specifying dependencies precisely since our builds fail due to random changes in the tools that we use.

Reply:
The whole point of having the version number appear in the dumped code (the version used to translate/dump the code) is so that it will continue to work for later versions of the compiler.  Those later versions will use a later version number for themselves, but when picking up the older code will get the older version in it.